requirements_dev.txt

```diff
@@ -36,6 +36,7 @@ lightgbm
 rdkit-pypi; sys_platform != 'darwin' or (sys_platform == 'darwin' and platform.machine != 'arm64')
 rdkit-pypi; python_version > '3.7' and sys_platform == 'darwin' and platform.machine == 'arm64'
 prometheus_client
+ultralytics; python_version >= '3.7' and sys_platform == 'linux'
```

Comment:
no need for this to be here. if you are only writing functional tests (i.e. yea tests) you could specify the requirements in the assertion file.

Reply:
:+1: removed this from the requirements_dev.txt file

Reply:
Thanks!